src/app/wallet-activation/project-activation/project-activation.component.ts

```diff
@@ -50,11 +52,17 @@ export class ProjectActivationComponent implements OnInit {
             this.broadService.broadcastSignedTx(sigRes.result.signedTransaction, res.tx_id)
                 .subscribe(_ => {
                     SpinnerUtil.hideSpinner();
-                    swal('', 'Success', 'success');
+                    swal('', 'Success', 'success').then(() => {
+                        this.reloadPage('/dash/activation/projects');
```

Comment:
This is an aggressive way of pulling fresh data to the screen and could be unnecessary for this scenario.
Why the following code wouldn't make it work?
```suggestion
                         this.fetchWalletToActivate();
```

Reply:
When I tested (several times) it didn't work, so I decided to reload the page.
Now when I try to call function again looks like it works. Maybe it's arkane issue.
However I suggest that we continue testing it

Reply:
Take some time to test it. This way should work. Page reload is considered as a bad practice and should be avoided (and used where really required). Tell me when you have a working version to test it out by myself.

Reply:
I made those changes on the last pull request, so you can test it when you have time! ;-)